### DISCUSSION TITLE
Error - Install 64 bits

### DISCUSSION THREAD

**Discussion by mauriciorocha1968** [skia4delphi/skia4delphi#314] [General]
Microsoft (R) VersÆo do Mecanismo de Compila‡Æo 4.8.9032.0
[Microsoft .NET Framework, VersÆo 4.0.30319.42000]
Copyright (C) Microsoft Corporation. Todos os direitos reservados.

Compila‡Æo de 09/05/2024 12:48:47 iniciada.
Projeto "C:\Componentes\Skia4Delphi\Packages\RAD Studio 12 Athens\Skia.Package.RTL.dproj" no n¢ 1 (destinos padrÆo).
CreateProjectDirectories:
  Criando diret¢rio "..\..\Library\RAD Studio 12 Athens\Win64\Release\".
  md "..\..\Library\RAD Studio 12 Athens\Win64\Release\"
  Criando diret¢rio "..\..\Library\RAD Studio 12 Athens\Win64\Include\".
  md "..\..\Library\RAD Studio 12 Athens\Win64\Include\"
  Criando diret¢rio "..\..\Library\RAD Studio 12 Athens\Win64\Release\Bpl\".
  md "..\..\Library\RAD Studio 12 Athens\Win64\Release\Bpl\"
C:\Program Files (x86)\Embarcadero\Studio\23.0\Bin\CodeGear.Delphi.Targets(425,5): error MSB6004: The specified task executable location "C:\Program Files (x86)\Embarcadero\Studio\23.0\bin\dcc64.exe" is invalid. [C:\Componentes\Skia4Delphi\Packages\RAD Studio 12 Athens\Skia.Package.RTL.dproj]
Done Building Project "C:\Componentes\Skia4Delphi\Packages\RAD Studio 12 Athens\Skia.Package.RTL.dproj" (default targets) -- FAILED.

FALHA da compila‡Æo.

"C:\Componentes\Skia4Delphi\Packages\RAD Studio 12 Athens\Skia.Package.RTL.dproj" (destino padrÆo) (1) ->
(_PasCoreCompile destino) -> 
  C:\Program Files (x86)\Embarcadero\Studio\23.0\Bin\CodeGear.Delphi.Targets(425,5): error MSB6004: The specified task executable location "C:\Program Files (x86)\Embarcadero\Studio\23.0\bin\dcc64.exe" is invalid. [C:\Componentes\Skia4Delphi\Packages\RAD Studio 12 Athens\Skia.Package.RTL.dproj]

    0 Aviso(s)
    1 Erro(s)

Tempo Decorrido 00:00:00.18

**Comment by checkdigits** [comment]
It looks like you might be trying to compile using the CE version of RAD Delphi (you don't mention the version in your bug report).

The CE version of Delphi12 doesn't include the command line compilers `DCC32.EXE` and `DCC64.EXE` so it will fail.

If you are not sure and you think you **should** have the command line compiler installed you can use "Everything" from Void Tools to search for it. It's free and you can find it here: [www.voidtools.com](https://www.voidtools.com)

If you are not using the CE version and you do find your `DCC64.EXE` in a different location please update this bug report with a comment listing your version, and a screenshot or note showing the location of DCC64.
